examples/erlang/esp32/sx127x.erl

```diff
@@ -67,17 +67,16 @@
 
 -define (SPISettings, [
     {bus_config, [
-            {miso_io_num, 19},
-            {mosi_io_num, 27},
-            {sclk_io_num, 5}
-        ]},
-        {device_config, [
-            {spi_clock_hz, 1000000},
-            {spi_mode, 0},
-            {spi_cs_io_num, 18},
-            {address_len_bits, 8}
+        {miso_io_num, 12},
+        {mosi_io_num, 13},
+        {sclk_io_num, 14}
+    ]},
+    {device_config, [
+        {my_device, [
+            {spi_cs_io_num, 18}
```

Comment:
are `spi_mode`, `spi_clock_hz`, `address_len_bits` still valid?

Reply:
Yes, they are.  I just removed them from the example program because the ones I removed are the default values and it makes the program simpler.